Salary App/Features/SalaryForm/Model/Calculators/NICalculator.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  NICalculator.swift
+//  Salary App
+//
+//  Created by Tobi Adegoroye on 12/07/2023.
+//
+
+import Foundation
+
+struct NICalaculator {
+    
+    func calculate(salary: Double) -> Double {
+       // £30,000 - £9,568 = £20,432
+      // (£20,432 * 0.12) / 12 = £204.32
+
+        let ni = salary - 9568
+        let total = (ni * 0.12)/12
+        let doubleStr = String(format: "%.2f", total) // "3.14"
```

Comment:
Why are you converting it into a string and then back into a double again? This is redundant since you just want to return the double anyways